util/install_deps_ubuntu.sh

```diff
@@ -32,20 +33,33 @@ deps=(
     libtool
 )
 
-# Ubuntu ARM64
-if [ "$(uname -m)" == "aarch64" ]; then
-    # For LAPACK
-    deps+=("gfortran")
+source /etc/lsb-release
```

Comment:
[nit] We can use eval to prevent polluting with unwanted variables from lsb-release.
https://unix.stackexchange.com/a/266630

Reply:
done.